solidity/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "stellar-bridge",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
@tamirms @nikhilsaraf Similar here, can we use the Apache 2.0 license?

Reply:
Yes we should use Apache 2.0 across the board